README.md

```diff
@@ -1,2 +1,841 @@
-# itn-did-spec
-ITN DID Method Specification
+The DID ITN Method Specification 1.0
+==================
+
+**Specification Status**: Working Draft
+
+**Latest Draft:** [ITN DID Specification](https://github.com/itn-trust/itn-did-spec)
+
+**Draft Created:** April 10, 2024
+
+**Latest Update:** April 11, 2024
+
+**Contributors & Editors:**
+
+1. [Andreas Freund](https://github.com/Therecanbeonlyone1969)
+2. [Dmitry Zotov](https://github.com/AlexMesser)
+3. [Parth Bhatt](https://github.com/bparth24)
+4. [Umed Khudoiberdiev](https://github.com/pleerock)
+
+
+## Abstract
+
+Integrated Trust Network (ITN) is a community-built and operated network. The objective is to build a Federated Trust Layer (Core Services) and Business Automation Interoperability Infrastructure (User Agents) for ITN. The ITN Identifier is specified in accordance with the [W3C DID Core specification](https://www.w3.org/TR/did-core/).
+
+ITN Identifier = ITN DID
+
+The syntax of the identifier and accompanying data model used by the protocol is conformant with the [DID-CORE](https://www.w3.org/TR/did-core/) specification and shall be registered with the [DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-spec-registries/).
+
+For a detailed read on DID and other DID method specifications, one can refer [WebOfTrustInfo - DID Primer](https://github.com/WebOfTrustInfo/rwot5-boston/blob/master/topics-and-advance-readings/did-primer.md)
+
+## Architecture Overview
+
+ITN Agent is an implementation of the [Aries Agent RFC](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0004-agents/README.md)
+with a flexible plugin architecture.
+
+Depending on which plugins and [DIDComm protocols](https://didcomm.org/search/?page=1) you use, your instance of ITN Agent can
+perform a variety of roles:
+
+* Create and manage cryptographic keys
+* Create and manage Decentralized Identifiers
+* Issue and verify Verifiable Credentials (VCs) and Presentations (VPs)
+* Communicate with other Agents over [DIDComm](https://identity.foundation/didcomm-messaging/spec/#discover-features-protocol-20)
+
+Agent provides a common interface for plugins to expand its functionality. It is designed to be modular and extensible,
+so you can add new protocols, transports, and managers to fit your needs.
+
+The agent was written in [TypeScript](https://www.typescriptlang.org/) and runs natively in Node.js, Browsers, and React Native.
+
+## ITN SDK
+
+![ITN_SDK](https://github.com/itn-trust/itn-did-spec/assets/18353464/6135b073-c085-41fd-8ea4-f80e64e0485d)
+
+One of the protocols is Identity Protocol provides methods to manage DIDs.
+One of the managers is DID Document Manager provide a set of functions to manage DID Documents.
+
+`Conirm with Umed & Andreas - here I am only providing necessary information related to DIDs & DID Document. Is that okay?`
```

Comment:
yes only related to the spec. "One of the protocols ...", and "One of the managers ..." is confusing better is"The ITN SDK's ...." instead of "One of ..."

Reply:
Noted and updated.